assets/initial_algorithms.json

```diff
@@ -0,0 +1,136 @@
+                    "LAST_NAME": "func:recordlinker.linking.matchers.feature_match_log_odds_fuzzy_compare"
+                },
+                "rule": "func:recordlinker.linking.matchers.eval_log_odds_cutoff",
+                "cluster_ratio": 0.9,
+                "kwargs": {
+                    "similarity_measure": "JaroWinkler",
+                    "thresholds": {
+                        "FIRST_NAME": 0.9,
+                        "LAST_NAME": 0.9,
+                        "BIRTHDATE": 0.95,
+                        "ADDRESS": 0.9,
+                        "CITY": 0.92,
+                        "ZIP": 0.95
+                    },
+                    "true_match_threshold": 12.2,
+                    "log_odds": {
+                        "ADDRESS": 8.438284928858774,
+                        "BIRTHDATE": 10.126641103800338,
+                        "CITY": 2.438553006137189,
+                        "FIRST_NAME": 6.849475906891162,
+                        "LAST_NAME": 6.350720397426025,
+                        "MRN": 0.3051262572525359,
+                        "SEX": 0.7510419059643679,
+                        "STATE": 0.022376768992488694,
+                        "ZIP": 4.975031471124867
+                    }
+                }
+            },
+            {
+                "blocking_keys": [
+                    "ZIP",
+                    "FIRST_NAME",
+                    "LAST_NAME",
+                    "SEX"
+                ],
+                "evaluators": {
+                    "ADDRESS": "func:recordlinker.linking.matchers.feature_match_log_odds_fuzzy_compare",
+                    "BIRTHDATE": "func:recordlinker.linking.matchers.feature_match_log_odds_fuzzy_compare"
+                },
+                "rule": "func:recordlinker.linking.matchers.eval_log_odds_cutoff",
+                "cluster_ratio": 0.9,
+                "kwargs": {
+                    "similarity_measure": "JaroWinkler",
+                    "thresholds": {
+                        "FIRST_NAME": 0.9,
+                        "LAST_NAME": 0.9,
+                        "BIRTHDATE": 0.95,
+                        "ADDRESS": 0.9,
+                        "CITY": 0.92,
+                        "ZIP": 0.95
+                    },
+                    "true_match_threshold": 17.0,
+                    "log_odds": {
+                        "ADDRESS": 8.438284928858774,
+                        "BIRTHDATE": 10.126641103800338,
+                        "CITY": 2.438553006137189,
+                        "FIRST_NAME": 6.849475906891162,
+                        "LAST_NAME": 6.350720397426025,
+                        "MRN": 0.3051262572525359,
+                        "SEX": 0.7510419059643679,
+                        "STATE": 0.022376768992488694,
+                        "ZIP": 4.975031471124867
+                    }
+                }
+            }
+        ]
+    }
+]
```

Comment:
It might be easier to reference these algorithms in tests, or other places, if we explicitly name them here.

```
basic_algorithm = { ... }
enhanced_algorithm = { ... }

algorithms = [basic_algorithm, enhanced_algorithm]
```

Reply:
Are you suggesting we make the top level entity in this JSON file an object rather than an array?

Reply:
Yeah. I forgot this file is a `.json` file so ignore my suggestion above. But basically making it easier to reference these algos instead of doing 
```
 for algo in load_json_asset("initial_algorithms.json"):
        if algo["label"] == "dibbs-basic":
            return models.Algorithm.from_dict(**algo)
```

But actually you only have to load these algorithms once in `the `conf.test` file so its probably not necessary to make any change here.

Reply:
Yeah, I used the [functools.lru_cache](https://docs.python.org/3/library/functools.html#functools.lru_cache) decorator to speed it up a bit as well.  That method should only have to load the json file and parse it once, all subsequent calls to the function should just use the cached result.